code/public/index.html

```diff
@@ -4,16 +4,12 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>Technigo React App</title>
+    <meta name="description" content="A site displaying all your favourite movies!" />
+    <meta name="keywords" content="Movie, Cinema, Now Playing, Popular, Top Rated, React">
+    <meta property="og:title" content="Movie Siter" />
+    <meta property="og:description" content="A site displaying all your favourite movies!" />
+    <meta property="og:url" content="https://project-movies-emelie.netlify.app/" />
```

Comment:
I guess you did not want to add the og:image. It can be a nice touch when the page is shared on social media.